<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>bbz Notenmanager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="nav">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>400</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QMdiArea" name="main">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_open_db"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="action_add_students"/>
    <addaction name="action_companies"/>
   </widget>
   <widget class="QMenu" name="menuErstellen">
    <property name="title">
     <string>Erstellen</string>
    </property>
    <addaction name="action_new_practice"/>
    <addaction name="action_new_theory"/>
    <addaction name="action_new_course"/>
    <addaction name="action_new_db"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuErstellen"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="status"/>
  <widget class="QToolBar" name="tools">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_db"/>
   <addaction name="separator"/>
   <addaction name="action_new_practice"/>
   <addaction name="action_new_theory"/>
   <addaction name="action_new_course"/>
   <addaction name="action_add_students"/>
   <addaction name="action_companies"/>
   <addaction name="separator"/>
   <addaction name="action_help"/>
  </widget>
  <action name="action_open_db">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/db-open</normaloff>:/icons/db-open</iconset>
   </property>
   <property name="text">
    <string>Öffnen</string>
   </property>
   <property name="toolTip">
    <string>Datenbank öffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_new_db">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/db-new</normaloff>:/icons/db-new</iconset>
   </property>
   <property name="text">
    <string>Neue Datenbank</string>
   </property>
   <property name="toolTip">
    <string>Neue Datenbank erstellen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_new_course">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/course-new</normaloff>:/icons/course-new</iconset>
   </property>
   <property name="text">
    <string>Neuer Kurs</string>
   </property>
   <property name="toolTip">
    <string>Neuen Kurs erstellen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_new_theory">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/theory</normaloff>:/icons/theory</iconset>
   </property>
   <property name="text">
    <string>Neuer Test</string>
   </property>
   <property name="toolTip">
    <string>Neuen Test anlegen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_new_practice">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/practice</normaloff>:/icons/practice</iconset>
   </property>
   <property name="text">
    <string>Neuer Versuch</string>
   </property>
   <property name="toolTip">
    <string>Neuen Versuch anlegen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_add_students">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/group-new</normaloff>:/icons/group-new</iconset>
   </property>
   <property name="text">
    <string>Teilnehmer bearbeiten</string>
   </property>
   <property name="toolTip">
    <string>Teilnehmer hinzufügen / bearbeiten</string>
   </property>
  </action>
  <action name="action_companies">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/new</normaloff>:/icons/new</iconset>
   </property>
   <property name="text">
    <string>Firmen bearbeiten</string>
   </property>
   <property name="toolTip">
    <string>Firmen hinzufügen / ändern</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/help</normaloff>:/icons/help</iconset>
   </property>
   <property name="text">
    <string>Dokumentation / Hilfe</string>
   </property>
   <property name="toolTip">
    <string>Dokumentation und Hilfe zum Notenmanager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/info</normaloff>:/icons/info</iconset>
   </property>
   <property name="text">
    <string>Über</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
